proto/ibc/applications/fee/v1/query.proto

```diff
@@ -51,5 +51,5 @@ message QueryIncentivizedPacketRequest {
 // QueryIncentivizedPacketsResponse is the response type for the incentivized packet RPC
 message QueryIncentivizedPacketResponse {
   // Incentivized_packet
-  ibc.applications.fee.v1.IdentifiedPacketFee incentivized_packet = 1;
+  ibc.applications.fee.v1.IdentifiedPacketFees incentivized_packet = 1 [(gogoproto.nullable) = false];
```

Comment:
This query doesn't make much sense to me. Why include packet id in response when it is already in request?

Makes more sense to have a PacketFee RPC that takes packetID and returns the packet fees for it.

Also I think it makes more sense to have an RPC that just aggregates all packet fees for a given packetID.

Something like

`QueryRecvFee(packetID)` => `total receive fees earned for this packet`

and similar for ack and timeout

Reply:
I totally agree, my main goal with this PR was to just fix what was there w.r.t. model changes that were made. 
I think we should definitely follow up on fleshing out the query api. I really like the idea of:
`QueryRecvFee(packetID)` => `total receive fees earned for this packet`
`QueryAckFee(packetID)` ...
`QueryTimeoutFee(packetID)` ...

If you want I can update this response type to include just `PacketFees` (the list of packet fees without the packet ID) instead of the identified type.